### PR TITLE
fix possibility to generate duplicate user with seed

### PR DESCRIPTION
**Pull request by mabiede**
- add a command for clean seeding to a specific tenant
- fix possibility to generate duplicate user with seed

### CODE REVIEW

File: pool/database/seed/seed_user.ml [hunk 1]
```diff
@@ -63,23 +63,51 @@ let create_rand_persons n_persons =
...
+      let%lwt iter =
+        n_persons - current_count
+        |> generate_persons
+        ||> flatten_filter_combine acc
+      in
+      persons_chunked iter n_persons)
+    else take_drop n_persons acc |> fst |> Lwt.return
   in
-  CCList.flatten persons_chunked |> Lwt.return
+  persons_chunked [] n_persons
```
Comment (timohuber) [inline pool/database/seed/seed_user.ml:104]: I think this function will always generate `n_persons` in one chunk, as `n_persons - current_count` in the first iteration is always `n_persons`? Or am I wrong?

```
  let rec persons_chunked acc chunk_size =
    let current_count = length acc in
    if current_count < n_persons
    then (
      Logs.info (fun m ->
        m "Seed: generating person data (%i/%i)" current_count n_persons);
      let%lwt iter =
        CCInt.min (n_persons - current_count) chunk_size
        |> generate_persons
        ||> flatten_filter_combine acc
      in
      persons_chunked iter chunk_size)
    else take_drop n_persons acc |> fst |> Lwt.return
  in
  persons_chunked [] chunk_size
```

Comment (mabiede) [reply pool/database/seed/seed_user.ml:104]: Yes, in the first run, it's usually always `n_persons`.
But `chunk_size` is needed for the API calls, which is handled by the `generate_persons` function.

Comment (mabiede) [reply pool/database/seed/seed_user.ml]: I removed the unneeded variable forwarding of `n_persons` and directly use the main function input.

Comment (timohuber) [reply pool/database/seed/seed_user.ml]: I tested my change before and it worked :) I passed `CCInt.min (n_persons - current_count) chunk_size` to generate_persons function.

But now the functionality is clearer.

**Review by timohuber — APPROVED** [review]

**Pull request merged**